sechub-sereco/src/test/java/com/mercedesbenz/sechub/sereco/WorkspaceTest.java

```diff
@@ -59,9 +73,36 @@ void do_import__no_importer() throws IOException {
         /* formatter:on */
 
         /* execute */
-        IOException exception = assertThrows(IOException.class, () -> workspace.doImport(importParameter));
+        IOException exception = assertThrows(IOException.class, () -> workspace.doImport(new SecHubConfiguration(), importParameter));
 
         /* test */
         assertEquals("Import failed, no importer was able to import product result: " + productId, exception.getMessage());
     }
+
+    @Test
+    void do_import_does_call_sensitive_data_masking_service() throws IOException {
+        /* prepare */
+        SecHubMessage info = new SecHubMessage(SecHubMessageType.INFO, "info");
+        String importId = "id1";
+        String importData = TestFileReader.loadTextFile("src/test/resources/sarif/sarif_2.1.0_owasp_zap.json");
+        String productId = "PDS_WEBSCAN";
+        List<SecHubMessage> messages = List.of(info);
+
+        /* formatter:off */
+        ImportParameter importParameter = ImportParameter.builder().importData(importData).importId(importId).importProductMessages(messages)
+                .productId(productId).build();
+        /* formatter:on */
+
+        List<ProductResultImporter> importers = new ArrayList<>();
+        importers.add(new SarifV1JSONImporter());
```

Comment:
Here the sarif importer is tested also.

I would like to have here a 
```java 
ProductResultImporter fakeImporter = mock(ProductResultImporter.class);
// maybe do some stuff with fakeImporter to provide wanted behavior
importers.add(fakeImporter);

```

The advantage will be:
- no additional dependency to sarif importer/no testing of other class
- it is clear that it not ony works with the sarif importer
- you do not need to load the textfile and the other data stuff before